genome_grist/conf/Snakefile

```diff
@@ -472,7 +472,7 @@ rule mpileup_wc:
         gunzip -c {input.query} > $genomefile
         bcftools mpileup -Ou -f $genomefile {input.bam} | bcftools call -mv -Ob -o {output.bcf}
         rm $genomefile
-        bcftools view {output.bcf} | bgzip > {output.vcf}
+        bcftools view {output.bcf} | b -9 > {output.vcf}
```

Comment:
I, uhh, does this actually work? I've never seen the `b` command and it doesn't exist on my laptop or on farm...

Reply:
Wait, what?! I've never seen that too :D Sorry it's a typo

Reply:
ok, but just to be clear, _you_ proposed this for merge, right? :)

Reply:
('MRG' in PR title)

Reply:
Yes! because there's no further in-progress work.

Reply:
@ctb fixed and ready for review.

Reply:
>Yes! because there's no further in-progress work.

Always a good idea to look at the diff one last time before sending for review 😁